include/irr/asset/IMeshBuffer.h

```diff
@@ -42,6 +42,13 @@ struct DrawElementsIndirectCommand_t
     uint32_t baseInstance;
 };
 
+struct DispatchIndirectCommand_t 
```

Comment:
I'd move this to IComputePipeline.h or make separate utility header and move all 3 there

Reply:
Probably separate header is best option

Reply:
@devshgraphicsprogramming what do you think?

Reply:
I second criss, this (all indirect structs) should move to `irr/asset/IPipeline.h`